dio/README.md

```diff
@@ -56,9 +56,8 @@ timeout, and custom adapters etc.
 
 ### Add dependency
 
-```yaml
-dependencies:
-  diox: ^replace-with-latest-version
+```console
+$ dart pub add diox
 ```
```

Comment:
So it would be:

You can use the command to add `diox` as a dependency with the latest stable version:
```console
$ dart pub add diox
```

Or you can manually add `diox` into the `dependencies` section in your `pubspec.yaml`:
```yaml
dependencies:
  diox: ^replace-with-latest-version
```

Reply:
👍

Reply:
Applied